index.js

```diff
@@ -45,6 +45,9 @@ async function onTrack(event, settings) {
     const endpoint = new URL(`${baseUrl}/capture/`)
     endpoint.searchParams.set('ts', event.timestamp)
 
+    const namespace = uuidv5.uuidv5('null', 'PostHog', true)
+    const uuid = uuidv5.uuidv5(namespace, event.messageId)
```

Comment:
`uuidv5` docs show this example for making a custom namespace: https://www.npmjs.com/package/uuidv5

(I don't think it's worth worrying about whether we can cache in global state or anything like that...)